po/ja.po

```diff
@@ -1316,44 +1316,70 @@ msgid ""
 "   and offer a solution, e.g., by showing people where to find the relevant\n"
 "   information in the standard library."
 msgstr ""
+"1. 資料に慣れておいてください。要点を強調するためにスピーカーノートが用意されています（内容の追加に"
+"ご協力ください！）。プレゼン時には、スクリーンを見やすい状態で保つために、スピーカーノートはポップ"
+"アップウィンドウで開いてください（スピーカーノートの横にある小さな矢印をクリック）。\n"
+"\n"
+"1. 予め日程を決めておいてください。講座は最低でも3日かかるため、2週間にわたって日程を組む事を推奨し"
+"ています。過去受講者によると、講座の間に数日ギャップを設ける事で内容が吸収しやすくなります。\n"
+"\n"
+"1. 十分な広さの部屋を確保しておいてください。15~25名程度のクラスを推奨しています。受講者にとって質"
+"問がしやすい人数であり、1人の講師が質問に答える時間も確保できる規模だからです。また、皆さんはPCで作"
+"業をする必要があるため、講師を含めた人数分の机を用意しておいてください。ライブコーディング形式での"
+"実施を想定しているため、講壇は不要です。\n"
+"\n"
+"1. 当日は少し早めに到着して準備をしてください。自分のPCで実行する`mdbook serve`から直接プレゼンを行"
+"う事を推奨します（[インストール手順][3]はこちら）。これにより、ページ切り替え時に遅延なしで最適なパ"
+"フォーマンスが得られます。また、PCを使用する事で、受講者や自分自身が見つけたタイプミスなども修正可"
+"能になります。\n"
+"\n"
+"1. 練習問題は個人か小さいグループで解いてください。回答をレビューする時間も含め、各練習問題に30~45"
+"分を費やします。受講者が行き詰まっているかどうか、何か質問があるかなど確認してください。複数の受講"
+"者が同じ問題で詰まっている場合、クラス全体に対してそれを共有し、解決策を提供してください。例えば、"
+"探している情報が標準ライブラリのどこにあるかを示す、など。"
 
 #: src/running-the-course.md:43
 msgid ""
 "That is all, good luck running the course! We hope it will be as much fun for\n"
 "you as it has been for us!"
-msgstr ""
+msgstr "以上です。運営頑張ってください！皆さんにとっても楽しい時間になりますように！"
 
 #: src/running-the-course.md:46
 msgid ""
 "Please [provide feedback][1] afterwards so that we can keep improving the\n"
 "course. We would love to hear what worked well for you and what can be made\n"
 "better. Your students are also very welcome to [send us feedback][2]!"
 msgstr ""
+"本講座の改善に向けて[フィードバック][1]をお願いします。うまくいった点や改善点について幅広くご意見お"
+"聞かせください。[受講者からのフィードバック][2]も歓迎しております！"
 
 #: src/running-the-course/course-structure.md:1
 msgid "# Course Structure"
 msgstr ""
 
 #: src/running-the-course/course-structure.md:5
 msgid "The course is fast paced and covers a lot of ground:"
-msgstr ""
+msgstr "本講座はペースが早く、多くの内容をカバーします："
 
 #: src/running-the-course/course-structure.md:7
 msgid ""
 "* Day 1: Basic Rust, ownership and the borrow checker.\n"
 "* Day 2: Compound data types,  pattern matching, the standard library.\n"
 "* Day 3: Traits and generics, error handling, testing, unsafe Rust."
 msgstr ""
+"* Day 1： Rustの基本、所有権と借用チェッカ\n"
```

Comment:
Perhaps, we should translate borrow checker as "借用チェッカー".  I am aware of that the Rust Book translation rules says "we should remove 'ー' in the end of a word, if the word has three syllables", and the rule is not applicable to the word.   'チェッカー' has two syllables, because the word has two vowel. I prefer to "チェッカー" than "チェッカ" because "チェッカ" looks slightly weird to me. What do you think?